webextensions/api/proxy.json

```
@@ -489,20 +489,23 @@
               "firefox": [
                 {
                   "version_added": "56",
-                  "version_removed": "71"
+                  "version_removed": "79"
```

I think this got over corrected. It was right to switch Android to 79, but desktop should stay at 71

```suggestion
                  "version_removed": "71"
```

---

Thanks @ddbeck. This is what we call in the trade, a screwup. Thanks for catching it.